pkg/operator/bugutil/format.go

```diff
@@ -14,10 +14,28 @@ func GetBugURL(b bugzilla.Bug) string {
 	return fmt.Sprintf("<https://bugzilla.redhat.com/show_bug.cgi?id=%d|#%d>", b.ID, b.ID)
 }
 
+// 2020-09-23T13:06:29Z
+
+func ParseChangeWhenString(v string) time.Time {
+	parsedTime, err := time.Parse("2006-01-02T15:04:05Z", v)
```

Comment:
isn't that some RFC 3339?

Reply:
maybe, I haven't dug deep, but tested this and it works :-)